plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedpanel/CallBack.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.ext.git.client.compare.changedpanel;
+
+import org.eclipse.che.ide.api.data.tree.Node;
+
+/**
+ * Callback for triggering actions in changed files panel.
+ */
+public interface CallBack {
```

Comment:
Node selection callback. But I'd propose to reuse existed components either `org.eclipse.che.ide.ui.smartTree.event.SelectionChangedEvent.SelectionChangedHandler` or `com.google.gwt.event.logical.shared.SelectionHandler`